.ci/pull-request-lts-config.yaml

```
@@ -0,0 +1,57 @@
+version: "2.1"
+
+dependencies: ./project-dependencies-quarkus.yaml
+
+pre: |
+  export BUILD_MVN_OPTS="${{ env.BUILD_MVN_OPTS }} -nsu -fae -e -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=120"
+  echo "BUILD_MVN_OPTS=${{ env.BUILD_MVN_OPTS }}"
+  export BUILD_MVN_OPTS_LTS="-Dversion.io.quarkus=999-SNAPSHOT"
+  echo "BUILD_MVN_OPTS_LTS=${{ env.BUILD_MVN_OPTS_LTS }}"
+
+default:
+  build-command:
+    current: mvn -e clean install ${{ env.BUILD_MVN_OPTS }}
+    upstream: mvn -e clean install -DskipTests -Dquickly ${{ env.BUILD_MVN_OPTS }}
```

skipTests not needed normally with `quickly` profile

```suggestion
    upstream: mvn -e clean install -Dquickly ${{ env.BUILD_MVN_OPTS }}
```